bitnami/kafka/README.md

```diff
@@ -0,0 +1,215 @@
+| `image.pullSecrets`              | Specify image pull secrets                                                                                 | `nil` (does not add image pull secrets to deployed pods)           |
+| `image.debug`                    | Specify if debug values should be set                                                                      | `false`                                                            |
+| `updateStrategy`                 | Update strategy for the stateful set                                                                       | `1`                                                                |
+| `replicaCount`                   | Number of Kafka nodes                                                                                      | `1`                                                                |
+| `config`                         | Configuration file for Kafka                                                                               | `nil`                                                              |
+| `allowPlaintextListener`         | Allow to use the PLAINTEXT listener                                                                        | `true`                                                             |
+| `listener`                       | The address the socket server listens on.                                                                  | `nil`                                                              |
+| `advertisedListener`             | Hostname and port the broker will advertise to producers and consumers.                                    | `nil`                                                              |
+| `brokerId`                       | ID of the Kafka node                                                                                       | `-1`                                                               |
+| `deleteTopicEnable`              | Switch to enable topic deletion or not.                                                                    | `false`                                                            |
+| `heapOpts`                       | Kafka's Java Heap size.                                                                                    | `-Xmx1024m -Xms1024m`                                              |
+| `logFlushIntervalMessages`       | The number of messages to accept before forcing a flush of data to disk.                                   | `10000`                                                            |
+| `logFlushIntervalMs`             | The maximum amount of time a message can sit in a log before we force a flush.                             | `1000`                                                             |
```

Comment:
Don't we want to group these parameters in a kafka group?

Reply:
The only reason why I didn't group these parameters is when someone uses the kafka chart as a dependency. 
In that case, in order to override a value from the parent chart would be sth like:

`--set kafka.kafka.brokerID`